pythonforandroid/recipes/argon2-cffi/__init__.py

```diff
@@ -3,7 +3,7 @@
 
 class Argon2Recipe(CompiledComponentsPythonRecipe):
     version = '20.1.0'
-    url = 'https://github.com/hynek/argon2-cffi/archive/master.zip'
+    url = 'url = 'git+https://github.com/hynek/argon2-cffi@{version}'
```

Comment:
that doesn't look quite right :)

Reply:
this is my first pr ever, sorry if it's a bit shabby.

Reply:
np. though you may want to do a rebase and squash all those intermediate commits when it's working.

Reply:
yeah okay I will do that